lib/phoenix_starter_web/views/user_view.ex

```diff
@@ -0,0 +1,20 @@
+defmodule PhoenixStarterWeb.UserView do
+  use PhoenixStarterWeb, :view
+  alias PhoenixStarterWeb.UserView
+
+  def render("index.json", %{users: users}) do
```

Comment:
We have the `index` view, and the function to get all users, but we don't have a route or the respective function in  `UserController`. Is it supposed to be "ignored" for now?

Reply:
I left as because it can be useful in the future maybe. I'd just leave it

Reply:
Or maybe remove it. I'll remove it.